config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
```

Comment:
I'd like to keep this version to stay as close to the recommended OSDK changes as possible. There might be a particular reason this version is being used. If we have a specific reason to use latest kube-rbac-proxy then it can be changed.